### DISCUSSION TITLE
Can i set the page-width for printing?

### DISCUSSION THREAD

**Discussion by erwin-van-maasakkers** [tailwindlabs/tailwindcss#8816] [Help] [answered]
I have Laravel with Tailwind and i try to print a page. Apparently the printed width is 640px of less.
Can i set the page-width so that it matched the window on my screen?

Or is there a better way to print pages?

**Comment by NovusOpus** [comment, marked as answer]
You can use the print modifier: https://tailwindcss.com/docs/hover-focus-and-other-states#print-styles

Example:
```HTML
<div class="w-full print:w-[640px]">
  <h1>Document Title</h1>
  <p>Some Stuff to print</p>
  <p class="hidden print:block">This print belongs to ACME CO. </p> <!-- This Element is hidden, But visible when printed -->
</div>
```

Classes with the `print:` modifier only apply when the document is being printed.

**Reply by erwin-van-maasakkers** [reply]
Thats it!! Thank you!!

**Reply by kokyaozhang** [reply]
Does this not work in local host? Every time i use that option the print options seems to not have any changes.

**Reply by NovusOpus** [reply]
> Does this not work in local host? Every time i use that option the print options seems to not have any changes.

Yes, it should work in Local. Just make sure you're in version `3.0` or newest. Since the print modifier was introduced with Tailwind CSS `3.0`.

If you're having problems with version 3.0 or newest , maybe share a minimal reproduction of your code in case there's a problem from your end. so we can help you troubleshooting.
